lib/Runtime/PlatformAgnostic/Platform/Common/Intl.cpp

```diff
@@ -455,21 +479,31 @@ namespace Intl
         return ret;
     }
 
-    bool ResolveLocaleLookup(_In_z_ const char16 *locale, _Out_ char16 *resolved)
+    bool IsLocaleAvailable(_In_z_ const char *locale)
     {
-        // TODO (doilij): implement ResolveLocaleLookup
-        resolved[0] = '\0';
+        int32_t countAvailable = uloc_countAvailable();
+        Assert(countAvailable > 0);
+        for (int i = 0; i < countAvailable; i++)
+        {
+            const char *candidate = uloc_getAvailable(i);
+            if (strcmp(locale, candidate) == 0)
+            {
+                return true;
+            }
+        }
```

Comment:
uloc really doesn't have a better way of querying for a locale in there supportedLocaleList? (I'd think we could at least binary search in that list rather than checking every single one). Can we ask them if there is a better way?